src/erlfmt_cli.erl

```diff
@@ -361,24 +395,27 @@ resolve_config(
         verbose = PreferVerbose,
         print_width = PreferWidth,
         pragma = PreferPragma,
-        out = PreferOut
+        out = PreferOut,
+        range = PreferRange
     },
     #config{
         verbose = DefaultVerbose,
         print_width = DefaultWidth,
         pragma = DefaultPragma,
-        out = DefaultOut
+        out = DefaultOut,
+        range = DefaultRange
     }
 ) ->
     #config{
         verbose = PreferVerbose orelse DefaultVerbose,
-        print_width = resolve_width(PreferWidth, DefaultWidth),
+        print_width = resolve_undefined(PreferWidth, DefaultWidth),
         pragma = resolve_pragma(PreferPragma, DefaultPragma),
-        out = resolve_out(PreferOut, DefaultOut)
+        out = resolve_out(PreferOut, DefaultOut),
+        range = resolve_undefined(PreferRange, DefaultRange)
     }.
 
-resolve_width(undefined, W) -> W;
-resolve_width(W, _) -> W.
+resolve_undefined(undefined, W) -> W;
```

Comment:
A nit, take it or leave it
But I preferred having verbose functions for each parameter

Reply:
Argl, it would trigger my DRY obsession!

Reply:
No problem, but just to explain where my review is coming from.
I prefer readability over DRY.
DRY is only important to me if it removes a large maintenance burden.

Reply:
Having two functions doing the same thing doesn't match my definition of readability :)
'Same things should share a same name, different things should have distinct names'
Yet, I don't have strong feelings for this one, so I've switched back to one function per parameter.